app/server/utils/credit_transfers.py

```diff
@@ -530,18 +532,19 @@ def transfer_credit_via_phone(send_phone, receive_phone, transfer_amount):
 
     transfer_amount = abs(transfer_amount)
 
-    send_user = models.User.query.filter_by(phone=send_phone).first()
+    send_user = User.query.filter_by(phone=send_phone).first()
     if send_user is None:
         return {'status': 'Fail', 'message': "Can't send from phone number: " + send_phone}
 
-    receive_user = models.User.query.filter_by(phone=receive_phone).first()
+    receive_user = User.query.filter_by(phone=receive_phone).first()
     if receive_user is None:
         return {'status': 'Fail', 'message': "Can't send to phone number: " + send_phone}
 
     if send_user.transfer_account.balance < transfer_amount:
         return {'status': 'Fail', 'message': "Insufficient Funds"}
 
-    transfer = make_payment_transfer(transfer_amount, token, send_user, receive_user)
+    # TODO(refactor): removed token as second argument since it doesn't exist... where was it before?
+    transfer = make_payment_transfer(transfer_amount, send_user, receive_user)
```

Comment:
this code was breaking because `token` doesn't exist. i didn't touch that code, don't know how it was working before

Reply:
weird, must have slipped through. my bad